test/variational/test_variational.py

```diff
@@ -0,0 +1,87 @@
+"""Tests for `pypesto.sample` methods."""
+
+import numpy as np
+import pytest
+from scipy.stats import kstest
+
+import pypesto.optimize as optimize
+from pypesto.variational import variational_fit
+
+from ..sample.test_sample import (
+    create_petab_problem,
+    gaussian_mixture_problem,
+    gaussian_problem,
+    rosenbrock_problem,
+)
+
+
+def variational_petab_problem():
+    # create problem
+    problem = create_petab_problem()
+
+    result = variational_fit(
+        problem,
+        n_iterations=100,
+        method="advi",
+        x0=np.array([3, -4]),
+    )
+    return result
```

Comment:
What is this used for? Do not see it in the tests anywhere else. If not used, would remove it.

Reply:
true, this was an old test I wrote, not necessary anymore